### PR TITLE
Region dropdown styles

### PR DESCRIPTION
**Pull request by jishaal**
Opening as draft for review + so I can test different devices with the preview build

### CODE REVIEW

File: app/components/RegionDropdown/RegionDropdown.module.scss [hunk 1]
```diff
@@ -1,53 +1,45 @@
...
-  @include h2;
   font-weight: 500;
   width: 100%;
   padding: 1rem 1.5rem 1rem 2rem;
   border-radius: 8px;
   cursor: pointer;
+  color: inherit;
 
   &:focus {
     background: rgba(255, 255, 255, 0.05);
```
Comment (andycarrell) [inline app/components/RegionDropdown/RegionDropdown.module.scss:34]: Can we drop default outline etc here?

Comment (jishaal) [reply app/components/RegionDropdown/RegionDropdown.module.scss:34]: Yeah we can, but do we have an alternative style, especially for light version

Comment (jishaal) [reply app/components/RegionDropdown/RegionDropdown.module.scss:34]: cc @finnclark

Comment (andycarrell) [reply app/components/RegionDropdown/RegionDropdown.module.scss:34]: Could do a simple "ring" using tailwind for now, use ring-current so it matches text?

https://tailwindcss.com/docs/ring-color

Comment (finnclark) [reply app/components/RegionDropdown/RegionDropdown.module.scss:34]: Should we avoid adding tailwind stuff if we're moving away from it @andycarrell ?

Comment (andycarrell) [reply app/components/RegionDropdown/RegionDropdown.module.scss:34]: As something to unblock this PR and be ready to ship I think it's fine? Also, we can then copy that styling if we're happy with it

**Review by finnclark — APPROVED** [review]
I'm happy to approve this - all good if I fiddle with styles in a new PR to suss the hover/focus + padding values etc afterwards?

**Comment by jishaal** [comment]
Yes please do you CSS magic 🙏🏼 🧙🏼 @finnclark

**Pull request merged**